File: sample/test/fails/test-fail-output-2.py
def harness(p):
    p.expect_exact("Unique number sequence")

    unique_numbers = []
    while p.expect([r"(\d+) ", "That's all what I got!"]) != 1:
        val = int(p.match.group(0))
        unique_numbers.append(val)

    assert len(unique_numbers) == len(set(unique_numbers)), \
           f'expected unique sequence! Got: {unique_numbers}'

File: sample/test/fails/test-fail-output-2.c
#include <stdio.h>

int main(int argc, char **argv)
{
	const int arr[] = {1, 2, 3, 4, 4, 5, 6};

	puts("Unique number sequence");
	for (int i = 0; i < sizeof(arr) / sizeof(arr[0]); i++) {
		printf("%d ", arr[i]);
	}
	puts("\nThat's all what I got!");
}
